cmd/main.go

```diff
@@ -15,7 +14,7 @@ func main() {
 	log := zerolog.New(os.Stdout).With().Logger().Level(zerolog.GlobalLevel())
 	ls := lockservice.NewSimpleLockService(log)
 
-	scfg := lockclient.SimpleConfig{
+	scfg := lockservice.SimpleConfig{
```

Comment:
Use a new function from the source.

Reply:
`SimpleNewConfig()` has been added to `lockservice` now.